src/containers/EditSubjectFrontpage/Subjectpage.tsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2020-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree. *
+ */
+
+import React, { FC, useEffect, useState } from 'react';
+import { match, Route, RouteComponentProps, Switch } from 'react-router';
+import { injectT } from '@ndla/i18n';
+// @ts-ignore
+import { OneColumn } from '@ndla/ui';
+import EditSubjectpage from './EditSubjectpage';
+import { TranslateType } from '../../interfaces';
+import CreateSubjectpage from './CreateSubjectpage';
+import Footer from '../App/components/Footer';
+
+interface Props {
+  t: TranslateType;
+  match: match;
+  location: RouteComponentProps['location'];
+}
+
+const Subjectpage: FC<Props> = ({ t, match, location }) => {
+  const [previousLocation, setPreviousLocation] = useState('');
+  const [isNewlyCreated, setNewlyCreated] = useState(false);
+
+  useEffect(() => {
+    /\/subjectpage\/(.*)\/new/.test(location.pathname)
+      ? setNewlyCreated(true)
+      : setNewlyCreated(false);
+    if (previousLocation !== location.pathname) {
+      setPreviousLocation(location.pathname);
+    }
+  }, []);
+
+  return (
+    <div>
+      <OneColumn>
+        <Switch>
+          <Route
+            //TODO: Hvordan skal pathen se ut?
+            path={`${match.url}/:subjectId/:subjectpageId/edit/:selectedLanguage`}
```

Comment:
Lurer på hvordan pathen for redigering skal se ut (med subjectid, subjectpageid etc.)

Reply:
Ta med dei parameterene du treng for å redigere fagsida og oppdatere taksonomi, så pathen kan sjå ut som du foreslår.